recipes/sensv/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+mkdir -pv $PREFIX/bin
+
+# download data files
+curl http://www.bio8.cs.hku.hk/sensv/data.tar.gz --output data.tar.gz
```

Comment:
Please put this in the `meta.yaml`:

    source:
      - url: https://github.com/HKU-BAL/SENSV/archive/v{{ version }}.zip
        sha256: e8544c106a3d88a399ec63ea829b9b03497e171814f175f54af601e5f655a9c3
      - url: http://www.bio8.cs.hku.hk/sensv/data.tar.gz
        sha256: something
        folder: data

Reply:
done